package.json

```
@@ -22,7 +22,7 @@
     "lint": "echo \"No linter specified yet\"",
     "test": "ava",
     "release": "semantic-release",
-    "develop": "ag https://raw.githubusercontent.com/asyncapi/generator/v1.0.0-rc.12/test/docs/dummy.yml ./ -o test/output --force-write --watch-template",
+    "develop": "ag https://raw.githubusercontent.com/asyncapi/generator/master/test/docs/dummy.yml ./ -o test/output --force-write --watch-template",
```

```suggestion
    "develop": "ag https://raw.githubusercontent.com/asyncapi/generator/v1.1.5/test/docs/dummy.yml ./ -o test/output --force-write --watch-template",
```

there is a certain level of risk that the version on master gets updated and before it is supported by html-template, someone can have issues

---

ok, i will rollback this

---

done